yb-voyager/cmd/importData_test.go

```diff
@@ -846,6 +853,636 @@ CREATE TABLE public.foo (
 	4. Import Data File tests
 */
 
+////=========================================
```

Comment:
shall we create  a different file for import-data-live tests? It would be easier to navigate